tests/common/models.py

```diff
@@ -113,6 +113,29 @@ def forward(self, x):
         return self.net(x)
 
 
+class EvenSimplerMLP(torch.nn.Module):
+
+    def __init__(self, num_features: int, device: str):
+        super().__init__()
+        fc1 = torch.nn.Linear(num_features, num_features, device=device, bias=False)
+        fc2 = torch.nn.Linear(num_features, num_features, device=device, bias=False)
+
+        self.module = torch.nn.Sequential(fc1, torch.nn.ReLU(), fc2)
+
+    def forward(self, x):
+        return self.module(x)
+
+
+class SimpleComposerMLP(ComposerClassifier):
+
+    def __init__(self, num_features: int, device: str, num_classes: int = 3):
+        fc1 = torch.nn.Linear(num_features, num_features, device=device, bias=False)
+        fc2 = torch.nn.Linear(num_features, num_features, device=device, bias=False)
+
+        net = torch.nn.Sequential(fc1, torch.nn.ReLU(), fc2)
+        super().__init__(num_classes=num_classes, module=net)
```

Comment:
why do we need two if both are basically the same? 

Also why do we need this over SimpleMLP?

Reply:
One is ComposrModel and one iss nn.Module. We want to support both

Reply:
Ah got it! Maybe nice to put a docstring on each saying use for (test composer/test pytorch raw) in tests?